contracts/ExitHandler.sol

```diff
@@ -6,9 +6,11 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
+/* solium-disable security/no-block-members */
+
 pragma solidity 0.4.24;
 
-import "openzeppelin-solidity/contracts/math/Math.sol";
+import "../node_modules/openzeppelin-solidity/contracts/math/Math.sol";
```

Comment:
why?

Reply:
to make code linter happy. Any problem with such a code?

Reply:
> @johannbarbie anything else here?
looks good